src/resmom_win/win/mom_mach.c

```diff
@@ -540,13 +540,10 @@ mem_sum(job *pjob)
 	GetSystemInfo(&si);
 
 	/* Get the number of processes embedded in the job */
-	if (!(ret = QueryInformationJobObject(pjob->ji_hJob,
-			JobObjectBasicAccountingInformation,
-			&ji, sizeof(ji), NULL))) {
-		log_errf(-1, __func__, "QueryInformationJobObject for %d failed to get number of processes", pjob->ji_hJob);
```

Comment:
QueryInformationJobObject this code is called from mom main thread among otehr places. It will be called even when there are no jobs as per my understanding. So we will be logging errors wrongly. Same for below.

Reply:
I think we call QueryInformationJobObject with a job handle. In-case if you noticed there is a scenario where we called this without a proper job, then we can get that validated from job id of that pjob.
Apart from this, if we look cput_job in mom_mach.c, there we do log the failure if the call fails.

Reply:
I guess the case would be when the job has no task running inside it. I think it might safest to leave this in DEBUG3